doc/releases/changelog-dev.md

```diff
@@ -216,38 +216,47 @@
   operation on our quantum circuits for both qubit and CV devices.
   [(#1829)](https://github.com/PennyLaneAI/pennylane/pull/1829)
 
-* For Hamiltonians whose eigenvalue frequency spectrum is known, `qml.gradients.get_shift_rule` is
-  a function that computes the generalized parameter shift rules for the time evolution.
+* Given an operator of the form :math:`U=e^{iHt}` where :math:`H` has commuting terms and known
+  eigenvalues (and thus known frequency spectrum),
```

Comment:
Maybe just say "is a hermitian operator whose eigenvalues we can extract" and leave it up to (future versions) of PL to define what this means?

I doubt there are more than 5 people out there who know what "commuting terms", or more severely, "frequency spectrum" means, since it refers to very particular pieces of research.